zerver/tests/test_signup.py

```diff
@@ -2439,3 +2440,25 @@ def test_login_under_subdomains(self):
         self.assertEqual(user_id, user_profile.id)
         self.assertEqual(response.status_code, 302)
         self.assertIn('http://zulip.testserver', response.url)
+class FollowupEmailTest(ZulipTestCase):
+    def followup_day2_email_test(self):
+        # type: () -> None
+        user_profile = self.example_user('hamlet')
+        # If the day the user joined is Sunday
+        user_profile.date_joined = datetime.datetime(2018, 1, 7, 1, 0, 0, 0, pytz.UTC)
+        self.assertEqual(followup_day2_email_delay(user_profile), datetime.timedelta(days=2, hours=-1))
+        # If the day the user joined is Tuesday
+        user_profile.date_joined = datetime.datetime(2018, 1, 2, 1, 0, 0, 0, pytz.UTC)
+        self.assertEqual(followup_day2_email_delay(user_profile), datetime.timedelta(days=2, hours=-1))
+        # If the day the user joined is Thursday
+        user_profile.date_joined = datetime.datetime(2018, 1, 4, 1, 0, 0, 0, pytz.UTC)
+        self.assertEqual(followup_day2_email_delay(user_profile), datetime.timedelta(days=1, hours=-1))
+        # If the day the user joined is Friday
+        user_profile.date_joined = datetime.datetime(2018, 1, 5, 1, 0, 0, 0, pytz.UTC)
+        self.assertEqual(followup_day2_email_delay(user_profile), datetime.timedelta(days=3, hours=-1))
+
+        # Let the timezone of user is 'America/Phoenix'. For Phoenix(America) UTC DST offset is −07:00
```

Comment:
This comment can be `Time offset of America/Phoenix is -07:00`
(I realize there's some weirdness with DST, but it's not important for this comment)